zcash_primitives/src/transaction/components.rs

```diff
@@ -193,7 +193,7 @@ pub struct OutputDescription {
     pub cv: edwards::Point<Bls12, Unknown>,
     pub cmu: Fr,
     pub ephemeral_key: edwards::Point<Bls12, Unknown>,
-    pub enc_ciphertext: [u8; 580],
+    pub enc_ciphertext: [u8; 584],
```

Comment:
We seem to expect `4` bytes in some places and `32` bits elsewhere, should this be uniform?

Reply:
perhaps, but this issue does not seem exclusive to the UDA fork. further investigation needed.

Reply:
I think everything in the circuit is bits and everything outside the circuit is bytes. I'm trying to label everything as I see it. I also changed ENC_CIPHERTEXT_SIZE (=584) and OUT_CIPHERTEXT_SIZE (=84) in note_encryption.rs to public and imported those constants to components.rs